### DISCUSSION TITLE
Window / frame margins?

### DISCUSSION THREAD

**Discussion by aykaramba** [lem-project/lem#824] [General]
Hi.

By default, Lem-SDL bumps the text up against all 4 edges of the editor.  Is there a global parameter that can be set to adjust the margins inside all windows / frames?

It would be nice to have 10 or 12 pixels margins so code looks properly laid out in the editor.

Thanks!

**Comment by cxxxr** [comment]
Hi, Thank you for the feedback!
Currently, this functionality is not available in lem, but it would certainly be useful, and I would like to make it happen.

**Reply by aykaramba** [reply]
Aha, neat!  No problem at all, was just curious.  I will continue exploring Lem to see how much I can learn.

Lem-SDL is already a very good text editor, thank you very kindly for this project!  Hopefully I can help one day.
